src/journal/views.py

```diff
@@ -2229,13 +2229,19 @@ def send_user_email(request, user_id, article_id=None):
             request.POST,
             request.FILES,
         )
-
+        log_dict = {
+            'level': 'Info',
+            'action_text': f'{request.user} sent an email to {user.full_name}',
+            'types': 'Email',
+            'target': article if article else user,
+        }
```

Comment:
shouldn't this be moved to below the check for `form.is_valid` and after the email is sent?